NUI/Core/NUIUtilities.m

```diff
@@ -76,5 +76,166 @@ + (NSString*)selector:(NSString*)selector withSuffix:(NSString*)suffix
     return selector;
 }
 
++ (BOOL)isHtml:(NSString *)text
```

Comment:
Maybe this method could fit better in String category (either method or dynamic property)?
(so we can use it anywhere else if needed :) )